<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateCardKeyDialog</class>
 <widget class="QDialog" name="GenerateCardKeyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Key Generation</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="expireLabel">
         <property name="text">
          <string>Expire Date</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="commentLabel">
         <property name="text">
          <string>Comment</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="emailLabel">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="emailEdit"/>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QCheckBox" name="nonExpireCheckBox">
         <property name="text">
          <string>Non Expire</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="commentEdit"/>
       </item>
       <item row="4" column="2">
        <widget class="QDateEdit" name="dateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="errLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tipLabel">
       <property name="text">
        <string>Warning: This Operation will overwrite keys which exist on card!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
